gef-ui/server/utils/mocks/mock-applications.js

```diff
@@ -38,7 +38,7 @@ const commonDealItems = {
   comments: [],
   ukefDecision: [],
   ukefDecisionAccepted: false,
-  exporter: { status: CONSTANTS.DEAL_STATUS.COMPLETED, details: [], validation: [] },
+  exporter: { status: CONSTANTS.DEAL_STATUS.COMPLETED, details: [], validation: [], companyName: 'test company' },
```

Comment:
This looks a bit weird - I'm not sure what the `details` and `validation` fields are

When visiting the cover end date page the `deal.exporter` object looks like
```
exporter: {
      status: 'Completed',
      accounts: [Object],
      companiesHouseRegistrationNumber: '03824658',
      companyName: 'SOFTWIRE TECHNOLOGY LIMITED',
      dateOfCreation: '1999-08-13',
      registeredAddress: [Object],
      industries: [Array],
      selectedIndustry: [Object],
      correspondenceAddress: null,
      smeType: 'Small',
      probabilityOfDefault: 10,
      isFinanceIncreasing: true,
      updatedAt: 1734602762999
    }
```

Reply:
@MarRobSoftwire Do you think I have to remove them? I don't know either

Reply:
I'm not sure - maybe leave it for now since other tests might rely on it. Its probabyl worth seeing where else this mock is used & if that uses the exporter data

Reply:
But happy to leave for now to avoid scope creeping